ansible/tasks/setup-system.yml

```diff
@@ -32,6 +32,11 @@
     update-alternatives --set ebtables /usr/sbin/ebtables-nft
     systemctl restart ufw
 
+- name: Create Sysstat log directory
+  file:
+    path: /var/log/sysstat
+    state: directory
```

Comment:
I don't think it's the placement of this clause, but the cleanup performed afterwards by `scripts/91-log_cleanup.sh` that's tripping it up.

Reply:
agree updated to create it after its getting nuked.